srw/direct_access.cpp

```diff
@@ -0,0 +1,205 @@
+/*
+ * 2014+ Copyright (c) Yandex
```

Comment:
Module comment blocks just hadn't been cleaned up at the time. Now they are done.
But I'm not comfortable with putting personal copyright strings in source files (those names and dates just add clutter in my opinion and also become obsolete quickly).
Maybe it is finally a time to have an AUTHORS file (as you proposed some time ago)?